# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1232210485 18000
#      Sat Jan 17 11:41:25 2009 -0500
# Node ID 4d884a01684634e5c45a47050a943f0e8e5c4bc9
# Parent  b01fef323c24150ee8bfd6d59c0a08c0e9334386
__quiver__.m: __quiver__.m: Fix auto-size of (x,y) coord.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2009-01-17  Steven Verstoep <isgoed@hotmail.com>
+
+	* plot/__quiver__.m: __quiver__.m: Fix auto-size of (x,y) coord.
+
 2009-01-16  Daniel J Sebald <daniel.sebald@ieee.org>
 
 	* plot/gnuplot_drawnow.m: Fix for handling options.
 
 2009-01-16  Daniel J Sebald  <daniel.sebald@ieee.org>
 
 	* plot/legend.m: Fix legend order for both horizontal and
 	vertical string cell.
diff --git a/scripts/plot/__quiver__.m b/scripts/plot/__quiver__.m
--- a/scripts/plot/__quiver__.m
+++ b/scripts/plot/__quiver__.m
@@ -35,19 +35,19 @@ function hg = __quiver__ (varargin)
   endfor
 
   ioff = 3;
   if (nargin < (6 + is3d) || firstnonnumeric < (6 + is3d))
     u = varargin{ioff++};
     v = varargin{ioff++};
     if (is3d)
       w = varargin{ioff++}
-      [x, y, z] = meshgrid (1:size(u,1), 1:size(u,2), 1:max(size(w)));
+      [x, y, z] = meshgrid (1:size(u,2), 1:size(u,1), 1:max(size(w)));
     else
-      [x, y] = meshgrid (1:size(u,1), 1:size(u,2));
+      [x, y] = meshgrid (1:size(u,2), 1:size(u,1));
     endif
     if (nargin >= ioff && isnumeric (varargin{ioff})
 	&& isscalar (varargin{ioff}))
       autoscale = varargin{ioff++};
     endif
   else
     x = varargin{ioff++};
     y = varargin{ioff++};
